clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -2471,11 +2471,14 @@ void CodeGenModule::SetLLVMFunctionAttributesForDefinition(const Decl *D,
     B.addAttribute(llvm::Attribute::StackProtectReq);
 
   if (!D) {
+    // HLSL functions must always be inlined
```

Comment:
nit: we do try have comments be complete sentences and puctuated.
```suggestion
    // HLSL functions must always be inlined.
```